ooniprobe/View/Informed consent/Onboarding2ViewController.m

```diff
@@ -0,0 +1,45 @@
+#import "Onboarding2ViewController.h"
+
+@interface Onboarding2ViewController ()
+
+@end
+
+@implementation Onboarding2ViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    //Constraint for iPhoneSE
```

Comment:
Can you add this comment to all pieces of code using this pattern?

Reply:
ok